### ISSUE TITLE
[Broken Website] Guardian.Report (Not loading if Dynamic mode enabled)

### ISSUE THREAD

**Issue by MrLinks75** [darkreader/darkreader#7290]
# Broken Website Report

## Website Address

https://guardian.report/

## Steps To Reproduce

1. Have dynamic mode enabled.
2. Go on guardian report.
3. Dark Reader will not allow the website to load, Splash screen in .svg seems to be tackled by Dark Reader
4. Website won't load and the page will crash on google chrome.
## Expected Behavior

![image](https://user-images.githubusercontent.com/31570259/140639960-494db7fd-c5b3-486f-887d-aefbeb748e99.png)

## Actual Behavior

Page wont load, stay blocked on splashscreen, developer console won't report any data.
## Screenshots

![image](https://user-images.githubusercontent.com/31570259/140639893-51056077-ffaa-4220-95ff-d136fe8ab3e6.png)
![image](https://user-images.githubusercontent.com/31570259/140639944-feb5673d-8fbe-4975-b86d-41f4bd8f75ab.png)

## System Information

OS : Windows 11 : Build 22000.258
Browser : Google Chrome - Version 95.0.4638.69 (Official Build) (64-bit)
Dark Reader Version : Version 5 Preview (4.9.40)

**Comment by Gusted** [comment]
Duplicate of #7266, install v4.9.41 when it is in the extension store.
